docs/api.md

```diff
@@ -394,54 +331,57 @@ curl "https://manage-courses-backend.herokuapp.com/api/v1/2019/subjects"
+address1               | Text               |                                                                                                                                                                                                                                                                                                                      | Address line 1
+address2               | Text               |                                                                                                                                                                                                                                                                                                                      | Address line 2
+address3               | Text               |                                                                                                                                                                                                                                                                                                                      | Town/City
+address4               | Text               |                                                                                                                                                                                                                                                                                                                      | County
+postcode               | Text               |                                                                                                                                                                                                                                                                                                                      | Postcode
+region_code            | Text               | 01 to 11                                                                                                                                                                                                                                                                                                             | 2-character string
+data_download_format   | Text               | `ASCII DATA`, `ASCII DATA and UNICODE DATA` or `Unicode data`                                                                                                                                                                                                                                                        |
+oustanding_decisions   | Text               | `Alphabetic`, `Application code`, `Course / Alphabetic`, `Course / Application code`, `Course / Learner Number`, `Faculty / Alphabetic`, `Faculty / Application code`, `Faculty / Course / Alphabetic`, `Faculty / Course / Application code`, `Faculty / Course / Learner code`, `Not required` or `Learner Number` |
```

Comment:
oustanding_decisions

?

Reply:
unchanged, I just reformatted the table, you could raise this separately